### DISCUSSION TITLE
Regarding Data

### DISCUSSION THREAD

**Discussion by indiatce** [PyPSA/PyPSA#810] [Q&A]
Hi,
I am trying to do a study for a particular area, which includes analysis of an existing power system grid with further planning for advancement, and exploring scenarios for renewable energy resources and hosting capacity analysis. In this regard, I am trying to acquire the data as follows;
1. Buses (name, v_nom, x,y(coordinates) ,v_mag_max,v_mag_min)
2. Lines(name, from bus, to bus, inductance, resistance, length of line, s_nom, s_max, s_min)
3. Generator (name, connected to the bus, capital coast, investment cost, cost of generation, efficiency, min_up time, min_down time, start up cost, shut down cost)
4. Load (name, connected to the bus, p_set)
5. Transformer ( name, from bus, to bus, x, r, s_nom, capital cost, transformer ratio, v_ang_min, v_ang_max)
6. Load_p_set (loads by hour/min)
7. generator_pmax_pu (generations by hour/min)

Can anyone please suggest me, what more data I require for doing the analysis of the existing network? The data corresponding to point 6 and 7, i.e load_p_set and generator_pmax_pu, will be required for hourly or minute basis? Do we need all the data for yearly basis to do the analysis? If possible, can I get an understanding with the duration of data for this type of study.

**Comment by euronion** [comment]
Hi there,

You're question is better suited for the Discord server here: https://discord.com/invite/AnuJBk23FU

Discussions/Issues here focus on the development of the PyPSA framework, not applications.
